src/scanner/scanner.py

```diff
@@ -146,15 +146,16 @@ def export(
                 df.to_parquet(file)
```

Comment:
with the move to duckdb I think we don't need to save the data as parquet anymore.

Reply:
I left there because the implementation was simple and it wouldn't affect the code in any way